packages/apollo-react-relay-duct-tape/src/storeObservation/nodeFromCacheFieldPolicy.ts

```diff
@@ -2,6 +2,8 @@ import { FieldReadFunction } from "@apollo/client";
 import { FragmentSpreadNode, FragmentDefinitionNode } from "graphql";
 import invariant from "invariant";
 
+const FRAGMENT_DOCUMENT_CACHE = new WeakMap();
```

Comment:
Just realised that this is missing a type-annotation, so a reminder for next time.